.golangci.yml

```diff
@@ -1,9 +1,6 @@
 service:
-  golangci-lint-version: 1.15.x # use the fixed version to not introduce new linters unexpectedly
+  golangci-lint-version: 1.16.x # use the fixed version to not introduce new linters unexpectedly
 run:
-  # default concurrency is a available CPU number
-  concurrency: 4
```

Comment:
This will blow-up circle machine. It has 128 vCPU.

Reply:
WHat does "blow up" mean? It seems to be working properly.

Reply:
In the past, it would cause OOMing since there is an overhead per worker. Circle lies about vCPU count since it only gives 4 vCPU but reports 128 vCPUs. It's the same reason why bazel blows up on circle since it allocates more workers than the machine can support.

Reply:
Last report:
```
bin/linters.sh: line 68: 24099 Killed                  golangci-lint run ./...
```
Looks like OOM.

Reply:
Yeah, I put back the concurrency setting.